src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompatibility/Logging/Suppression.cs

```diff
@@ -13,6 +13,11 @@ public class Suppression : IEquatable<Suppression>
         /// </summary>
         public string DiagnosticId { get; set; }
 
+        /// <summary>
+        /// The diagnostic message describing the suppression.
+        /// </summary>
+        public string Message { get; set; }
```

Comment:
Do we want an option for folks to exclude `Message` if they want to keep the old file format?

Reply:
I wasn't planning to provide such an option. We can always go back and add a knob if we receive feedback.